jorvik/celery.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+
+import os
+import traceback
+from celery import Celery, group
+from celery.utils.log import get_task_logger
+from django.db import transaction
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'jorvik.settings')
+
+app = Celery('jorvik')
```

Comment:
Possiamo dare un nome piu' significativo a questa coda? Abbiamo tante cose che potrebbero beneficiare dall'esecuzione asincrona in un task Celery, quindi sarebbe importante separare questa coda dandogli un nome che indica che il suo obiettivo e' quello di smistare la coda (`jorvik_posta`?).

Reply:
OK.